flink-table/flink-sql-client/src/test/java/org/apache/flink/table/client/cli/CliClientTest.java

```diff
@@ -258,6 +258,31 @@ public void testUseDatabaseAndShowCurrentDatabase() throws Exception {
         assertTrue(output.contains("db"));
     }
 
+    @Test
+    public void testUseDatabaseAndShowCurrentDatabaseByKeyword() throws Exception {
+        TestingExecutor executor = new TestingExecutorBuilder()
+                .setExecuteSqlConsumer((ignored1, sql) -> {
+                    if (sql.toLowerCase().equals("use `mod`")) {
+                        return TestTableResult.TABLE_RESULT_OK;
+                    } else if (sql.toLowerCase().equals("show current database")) {
+                        SHOW_ROW.setField(0, "`mod`");
+                        return new TestTableResult(ResultKind.SUCCESS_WITH_CONTENT,
+                                TableSchema.builder().field("current database name", DataTypes.STRING()).build(),
+                                CloseableIterator.ofElement(SHOW_ROW, ele -> {}));
+                    } else {
+                        throw new SqlExecutionException("unexpected database name: db");
+                    }
+                })
+                .build();
+        String output = testExecuteSql(executor, "use `mod`;");
+        assertThat(executor.getNumExecuteSqlCalls(), is(1));
+        assertFalse(output.contains("OK"));
```

Comment:
Should be `assertFalse(output.contains("unexpected database name"));`.

This test can pass even if we don't fix `CliClient`.